testing/auth.ts

```diff
@@ -229,25 +230,32 @@ class CredentialHandler {
     print('Credentials updated!');
   }
 
-  handleOAuth2() {
-    assertCondition(this._authDef.type === AuthenticationType.OAuth2);
-    const existingCredentials = this.checkForExistingCredential() as OAuth2Credentials | undefined;
-    print(
-      `*** Your application must have ${makeRedirectUrl(this._oauthServerPort)} whitelisted as an OAuth redirect url ` +
-        'in order for this tool to work. ***',
-    );
+  private _promptOAuth2ClientIdAndSecret(existingCredentials: OAuth2Credentials | OAuth2ClientCredentials | undefined):
+      {clientId: string, clientSecret: string} {
     const clientIdPrompt = existingCredentials
-      ? `Enter the OAuth client id for this Pack (or Enter to skip and use existing):\n`
-      : `Enter the OAuth client id for this Pack:\n`;
+        ? `Enter the OAuth client id for this Pack (or Enter to skip and use existing):\n`
+        : `Enter the OAuth client id for this Pack:\n`;
     const newClientId = promptForInput(clientIdPrompt);
     const clientSecretPrompt = existingCredentials
-      ? `Enter the OAuth client secret for this Pack (or Enter to skip and use existing):\n`
-      : `Enter the OAuth client secret for this Pack:\n`;
+        ? `Enter the OAuth client secret for this Pack (or Enter to skip and use existing):\n`
+        : `Enter the OAuth client secret for this Pack:\n`;
     const newClientSecret = promptForInput(clientSecretPrompt, {mask: true});
 
     const clientId = ensureNonEmptyString(newClientId || existingCredentials?.clientId);
     const clientSecret = ensureNonEmptyString(newClientSecret || existingCredentials?.clientSecret);
 
+    return {clientId, clientSecret};
+  }
+
+  handleOAuth2() {
+    assertCondition(this._authDef.type === AuthenticationType.OAuth2);
+    const existingCredentials = this.checkForExistingCredential() as OAuth2Credentials | undefined;
+    print(
+      `*** Your application must have ${makeRedirectUrl(this._oauthServerPort)} whitelisted as an OAuth redirect url ` +
```

Comment:
Nit, this was already here, but mind fixing up to use `allowlisted` per inclusive language guidelines?